.github/workflows/manual.yml

```diff
@@ -0,0 +1,40 @@
+name: actions
+
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - 'doc/**'
+      - 'nixos/doc/**'
+      - 'nixos/**/*.xml'
+
+jobs:
+
+  nixpkgs-manual:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: technote-space/get-diff-action@v1.2.8
+        id: git_diff
+        with:
+          PREFIX_FILTER: doc
+      - uses: cachix/install-nix-action@v8
+        if: steps.git_diff.outputs.diff
+      - name: build nixpkgs manual
+        if: steps.git_diff.outputs.diff
+        run: nix-shell doc/shell.nix --run 'make -C doc'
```

Comment:
According to the manual https://nixos.org/nixpkgs/manual/#chap-contributing we build it with `nix-build ./doc`. However it also says that `make -C doc debug` might come in handy in case there are errors.

Reply:
cc @grahamc

Reply:
I hadn't noticed that `ofborg` is running [instantiate](https://github.com/NixOS/ofborg/blob/released/ofborg/src/tasks/eval/nixpkgs.rs#L480-L502) for the manuals, `nixpkgs` has `pkgs/top-level/release.nix -A manual`.